src/app/pages/ArticlePage/PagePromoSections/RelatedContentSection/index.jsx

```diff
@@ -136,7 +140,7 @@ const RelatedContentSection = ({ content }) => {
     >
       <SectionLabel
         labelId={LABEL_ID}
-        backgroundColor={C_GREY_2}
+        backgroundColor={GREY_2}
```

Comment:
Herem would it be better to create a 'StyledSectionLabel' styled component with a background-color declaration. Or is it better to keep it as it is, and pass in the background color as a prop?